Central and Eastern European Online Library Journals.js

```diff
@@ -0,0 +1,321 @@
+				"issue": "4",
+				"language": "Hungarian",
+				"libraryCatalog": "www.ceeol.com",
+				"pages": "63-76",
+				"publicationTitle": "Erdélyi Jogélet",
+				"url": "https://www.ceeol.com/search/article-detail?id=925530",
+				"volume": "III",
+				"attachments": [
+					{
+						"title": "Full Text PDF",
+						"mimeType": "application/pdf"
+					}
+				],
+				"tags": [
+					{
+						"tag": "artificial intelligence"
+					},
+					{
+						"tag": "automation"
+					},
+					{
+						"tag": "occupational safety"
+					},
+					{
+						"tag": "responsibility"
+					},
+					{
+						"tag": "robotics"
+					}
+				],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://www.ceeol.com/search/article-detail?id=934330",
+		"items": [
+			{
+				"itemType": "journalArticle",
+				"title": "Pavao Maček, Pogledići i Malenići. Dva plemenita roda od Kurilovca: S priloženim rodoslovnim stablima",
+				"creators": [
+					{
+						"firstName": "Ivan",
+						"lastName": "Jurković",
+						"creatorType": "author"
+					}
+				],
+				"date": "2020",
+				"ISSN": "0351-9767, 1848-9087",
+				"abstractNote": "Summary/Abstract:  Book-Review: Pavao Maček, Pogledići i Malenići. Dva plemenita roda od Kurilovca: S priloženim rodoslovnim stablima, Zagreb: Družtvo za povjestnicu Zagrebačke nadbiskupije “Tkalčić”, 2019, 574 stranice i dva rodoslovna stabla. Review by Ivan Jurković.",
+				"issue": "58",
+				"language": "Croatian",
+				"libraryCatalog": "www.ceeol.com",
+				"pages": "154-156",
+				"publicationTitle": "Povijesni prilozi",
+				"shortTitle": "Pavao Maček, Pogledići i Malenići. Dva plemenita roda od Kurilovca",
+				"url": "https://www.ceeol.com/search/article-detail?id=934330",
+				"attachments": [
+					{
+						"title": "Full Text PDF",
+						"mimeType": "application/pdf"
+					}
+				],
+				"tags": [
+					{
+						"tag": "Pavao Maček"
+					},
+					{
+						"tag": "Pogledići and Malenići"
+					},
+					{
+						"tag": "genealogical trees"
+					},
+					{
+						"tag": "two noble families from Kurilovac"
+					}
+				],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://www.ceeol.com/search/journal-detail?id=1266",
+		"defer": true,
+		"items": [
+			{
+				"itemType": "journalArticle",
```

Comment:
This still isn't a journal article, right? It should be a multiple.

Reply:
changed the css selector for detecting multiple downloads and updated the tests.